src/main/webapp/signup-test.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
If this is a file you created to test locally, please do not submit it.

Reply:
Understood, I will remove my test html files from all PR into user-feature. I plan to replace them with Angela's HTML soon.